utils/helpers/account/useGoogleIdentityServices.ts

```diff
@@ -65,10 +72,12 @@ export default function useOneTapSignin(
     );
     if (error) {
       logError("logInWithGoogle", error);
-      if (error.code === 401) {
+      if (error.code === 404) {
         plausible("Account Not Found");
         onNotFound?.();
-      }
+      } else
+        setSnackbar(createElement(Snackbar, undefined, tx("snackbar.failure")));
+      plausible("Log in Failed");
```

Comment:
This is good for making it easier to diagnose the problem and give tech support because there is a difference in the indication for the account not found problem and other problems.